dev/com.ibm.ws.jaxrs.2.0_fat/publish/servers/com.ibm.ws.jaxrs.fat.beanvalidation/bootstrap.properties

```diff
@@ -0,0 +1,8 @@
+com.ibm.ws.logging.trace.specification=*=info:\
+com.ibm.ws.jaxrs20.*=all:\
+com.ibm.websphere.jaxrs20.*=all:\
+com.ibm.ws.security.java2sec.*=all=enabled:\
+com.ibm.ws.classloading.java2sec.*=all=enabled
```

Comment:
Same as above about excess logging.
Also, we should probably remove `com.ibm.ws.security.java2sec.*=all=enabled:\` and `com.ibm.ws.classloading.java2sec.*=all=enabled` - my guess is that they were only used for a specific Java 2 security issue that is (hopefully) now resolved.